website/docusaurus.config.js

```diff
@@ -0,0 +1,148 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+import {themes as prismThemes} from 'prism-react-renderer';
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Chisel',
+  tagline: 'Software-defined hardware',
+  favicon: 'img/chisel-tool-icon.svg',
```

Comment:
I can't recall if this was changed here or earlier. However, "Chisel" is the correct name now as opposed to "Chisel/FIRRTL Hardware Compiler Framework" given the advent of MFC. 👍